shortest_common_superstring/shortest_common_superstring.py

```diff
@@ -16,6 +16,12 @@ def exact(T):
         return superword
     superwords = [superword + c for superword in superwords for c in A]
 
+def naive(T):
+  '''Does not change the order of words, only overlaps them'''
+  return '#' + ''.join(w1[1:-prefix.get_overlap(w1,w2)]
+      if prefix.get_overlap(w1,w2)>0 else w1[1:] for w1,w2 in zip(T,T[1:])) \
+          + T[-1][1:] if T else ''
```

Comment:
Please re-write it in 2-3 commands (e.g. finding W + return '#'.join(W)), this way it's somewhat messy